### DISCUSSION TITLE
Manage argo-cd projects' manifests via helm chart

### DISCUSSION THREAD

**Discussion by ep4sh** [argoproj/argo-cd#13698] [General]
Dear community,
as far as I could understand it, there is no chance to manage projects' settings via helm, I am very interested in:
`appproject.argoproj.io` resources, since I have multiple. 

My guess is to create it via UI and then dump to the IaC manifest

Please advise the proper way =)

**Comment by mubarak-j** [comment]
We manage ArgoCD projects using [argocd-apps](https://github.com/argoproj/argo-helm/tree/main/charts/argocd-apps) chart.
It has a [good example](https://github.com/argoproj/argo-helm/blob/cd4c291e47f0e9c1184fb05e79d9f3b742195c5c/charts/argocd-apps/values.yaml#L46-L87) commented out the in the values file.

**Reply by ep4sh** [reply]
thanks a lot, @mubarak-j !
